pkg/client/list.go

```diff
@@ -103,8 +103,9 @@ func GetDriveList(ctx context.Context, nodes, drives, accessTiers []utils.LabelV
 
 // ListVolumeResult denotes list of volume result.
 type ListVolumeResult struct {
-	Volume directcsi.DirectCSIVolume
-	Err    error
+	Volume   directcsi.DirectCSIVolume
+	Continue string
```

Comment:
We need `Continue` because the ListVolumes controller RPC response requires this continue token to be passed. This token sent in the response will be used for the next request.